.github/workflows/examples_component.yml

```diff
@@ -30,7 +30,7 @@ jobs:
   check:
     runs-on: ubuntu-latest
     outputs:
-      run-build: ${{ steps.force.outputs.force || steps.check-example-changes.outputs.changed_keys.components }}
+      run-build: ${{ steps.force.outputs.force || steps.check-example-changes.outputs.all_changed_files_count > 0 }}
```

Comment:
Though it seems worded a little strangely, most of the examples I've seen use [`any_changed`](https://github.com/tj-actions/changed-files?tab=readme-ov-file#output_any_changed) option to determine whether anything changed, for example: https://github.com/linkerd/linkerd2/blob/b2ae3f49e30c7c74b0bf33e50957c76266fd56fb/.github/workflows/go.yml#L24-L25

Reply:
Yes! I thought there was one like that but missed it in the debug output.